packages/amplify_api_plugin_interface/lib/src/GraphQLOperation.dart

```diff
@@ -0,0 +1,31 @@
+/*
+ * Copyright 2020 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License").
+ * You may not use this file except in compliance with the License.
+ * A copy of the License is located at
+ *
+ *  http://aws.amazon.com/apache2.0
+ *
+ * or in the "license" file accompanying this file. This file is distributed
+ * on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+ * express or implied. See the License for the specific language governing
+ * permissions and limitations under the License.
+ */
+
+import 'package:flutter/foundation.dart';
+import './types.dart';
+
+class GraphQLOperation<T> {
+  Function _cancel;
+
+  GraphQLOperation({@required this.response, @required Function cancel}) {
+    _cancel = cancel;
+  }
+
+  Future<GraphQLResponse<T>> response;
+
+  void cancel() {
+    _cancel();
```

Comment:
Move the member fields together at the top

Reply:
I'm slightly hesitant about this type. I think it's possible that customers could see `GraphQLOperation` and expect something more [akin to this.](https://github.com/zino-app/graphql-flutter/blob/a3e4401c09aa36f11a2f341f759a18702b32516d/packages/graphql/lib/src/link/operation.dart). Perhaps you could share type that Kyle is using to support cancel (I'm going off memory, so that might be off base).

Reply:
Updated.
Regarding the type name, it is inline with what we decided in the API review, and is also similar to what customers selected as their preferred DX in the Rest API RFC.